### ISSUE TITLE
Add ignoreTypes to value-keyword-case

### ISSUE THREAD

**Issue by havenchyk** [stylelint/stylelint#5166]
> What is the problem you're trying to solve?

I'm trying to fix the issue with styled-components that are checked by stylelint

> What solution would you like to see?

I want this property (*ignoreTypes*) to be added to the rule

>  ✖  Expected "dummyValue" to be "dummyvalue"   value-keyword-case

for code

```
const a = css`
  ${b};
  text-align: right;
`
```

Related issue https://github.com/styled-components/stylelint-config-styled-components/pull/25#issuecomment-654268058

**Comment by jeddy3** [comment]
@havenchyk Thanks for the request and for using the template.

Does the following not work for you?:

```js
"value-keyword-case": [
  "lower",
  {
    "ignoreKeywords": ["$dummyValue"]
  }
]
```

Notice that it's [`ignoreKeywords`](https://stylelint.io/user-guide/rules/value-keyword-case#ignorekeywords-regex-regex-non-regex) and not `ignoreTypes` as in the linked comment.

**Comment by havenchyk** [comment]
@jeddy3 thanks for the quick response! Actually I tried that option and it didn't work for me, but let me check once again

**Comment by jeddy3** [comment]
Sorry, I meant:

```json
{
  "processors": ["stylelint-processor-styled-components"],
  "extends": [
    "stylelint-config-standard",
    "stylelint-config-styled-components"
  ],
  "rules": {
    "value-keyword-case": [
      "lower",
      {
        "ignoreKeywords": ["dummyValue"]
      }
    ]
  }
}
```

I don't think the processor injects a dollar symbol in there.

**Comment by havenchyk** [comment]
it worked, thanks @jeddy3 🙇
